src/main/tv/codely/scala_http_api/module/user/infrastructure/dependency_injection/UserModuleDependencyContainer.scala

```diff
@@ -1,7 +1,10 @@
 package tv.codely.scala_http_api.module.user.infrastructure.dependency_injection
 
-import tv.codely.scala_http_api.module.user.application.UsersSearcher
+import tv.codely.scala_http_api.module.user.application.search.UsersSearcher
+import tv.codely.scala_http_api.module.user.infrastructure.repository.InMemoryUserRepository
 
 final class UserModuleDependencyContainer {
-  val usersSearcher = new UsersSearcher
+  val repository = new InMemoryUserRepository
+
+  val usersSearcher = new UsersSearcher(repository)
```

Comment:
We're using different naming conventions here for each one of these values:

If we want to be consistent, we could go for the:
```scala
val userRepository = new InMemoryUserRepository
val usersSearcher = new UsersSearcher(userRepository)
```

or the:
```scala
val repository = new InMemoryUserRepository
val searcher = new UsersSearcher(repository)
```

Use the one you prefer, I'm ok with both of them. But it would be nice to use the very same convention 🙂

Reply:
It's because of:
```scala
val repository = new InMemoryUserRepository

// We need to denote are not the same
val usersSearcher = new UsersSearcher(repository)
val userSearcher  = new UserSearcher(repository)
```